lib/components/form/LinkInverters.tsx

```diff
@@ -33,7 +34,7 @@ const SupportedInverters = () => {
             <img
```

Comment:
do we want to replace this with Image? either way i think i got this in main too with last pr lol

Reply:
I was trying before pushing this and I think it'll require another div and just a bit more junk, can do after showcase tomorrow